src/ddmd/declaration.h

```diff
@@ -523,6 +524,8 @@ class FuncDeclaration : public Declaration
     CompiledCtfeFunction *ctfeCode;     // Compiled code for interpreter
     int inlineNest;                     // !=0 if nested inline
     bool isArrayOp;                     // true if array operation
+    bool eh_none;                       /// true if no exception unwinding is needed
```

Comment:
Mmh, that doesn't belong in the frontend, any other place you could store or retrieve this flag?

Reply:
The flag is per-function, and is detected by the font end. It is later transferred to the backend's symbol for that same function.